src/workerd/io/worker.c++

```diff
@@ -2754,9 +2754,19 @@ void Worker::Actor::shutdown(uint16_t reasonCode) {
     // written.
   }
 
+  shutdownActorCache(error);
```

Comment:
Just wondering -- in the cases where `error` is null, would it be at all useful to pass down reasonCode so that `ActorCache::shutdown` at least has some sort of info to include in the exception it generates?

Reply:
For better or worse, the set of reasonCodes that lack an associated error are pretty uninformative at the moment: Either "unknown reason" or "evicted".